### DISCUSSION TITLE
Land Cover Error

### DISCUSSION THREAD

**Discussion by mitchaspinall** [UMEP-dev/UMEP#335] [Q&A]
Hi There, 
I seem to be having an issue with my land cover data. I think I know why the issue is occurring but I don't know how to fix it.

![image](https://user-images.githubusercontent.com/93663727/140079781-f457c7fa-9d07-4c27-ba8f-4f549db99a47.png)

I get that error every time I input my land cover data. Now I know that the values are meant to be 1 to 7, but when I look at my land cover raster that I generated in the preprocessor, I get these values:

Band 1	
STATISTICS_MAXIMUM=7
STATISTICS_MEAN=1.1930697451599
**STATISTICS_MINIMUM=0**
STATISTICS_STDDEV=1.7066576496392

So my question is, does anyone know how to change the minimum from 0 so that the processor recognizes the data?

Thanks so much

**Comment by biglimp** [comment]
Somewhere you have pixels with value 0. You need to identify them and reclass them to numbers between 1 and 7
